### ISSUE TITLE
Add automated accessibility testing

### ISSUE THREAD

**Issue by msecret** [18F/cg-dashboard#787]
As we've quickly developed the site, we've had some trouble ensuring we don't introduce new accessibility issues.

In order for the dev team to not contribute new accessibility issues, there should be automated accessibility tests that run as part of the build

_Most automated accessibility testing tools categorize potential problems as `error`, `warning`, and `notice`. The `errors` are almost always real, important problems, `warning`s can sometimes be problems but are often false alarms and `notice`s are usually not important._

**Comment by tessereth** [comment]
In case it's useful to you, my team has used both https://github.com/pa11y/pa11y and the ruby wrapper for https://github.com/dequelabs/axe-core. We found axe-core to be more comprehensive than pa11y and we only had one false positive from axe-core over 4 months. There's also https://github.com/reactjs/react-a11y, which I've only looked at briefly but it's probably worth a closer look.
